### ISSUE TITLE
Training - loss = nan

### ISSUE THREAD

**Issue by burgetrm** [balancap/SSD-Tensorflow#113]
I have created the training tfrecord from VOC2007 using this script:
```
OUTPUT_DIR=./tfrecords
DATASET_DIR=./VOC2007/train/
python tf_convert_data.py \
    --dataset_name=pascalvoc \
    --dataset_dir=${DATASET_DIR} \
    --output_name=voc_2007_train \
    --output_dir=${OUTPUT_DIR}
```

For the further training I have used the following code:

```
DATASET_DIR=./tfrecords
TRAIN_DIR=./logs/
CHECKPOINT_PATH=./checkpoints/ssd_300_vgg.ckpt
python3 train_ssd_network.py \
    --train_dir=${TRAIN_DIR} \
    --dataset_dir=${DATASET_DIR} \
    --dataset_name=pascalvoc_2007 \
    --dataset_split_name=train.
    --model_name=ssd_300_vgg \
    --checkpoint_path=${CHECKPOINT_PATH} \
    --save_summaries_secs=60 \
    --save_interval_secs=600 \
    --weight_decay=0.0005 \
    --optimizer=adam \
    --learning_rate=0.001 \
    --batch_size=32
```

Unfortunately during the training I got the loss function is evaluated as a not a number:
```
INFO:tensorflow:Recording summary at step 1.
INFO:tensorflow:global step 10: loss = 7524311953285637472256.0000 (1.586 sec/step)
INFO:tensorflow:global step 20: loss = nan (1.586 sec/step)
INFO:tensorflow:global step 30: loss = nan (1.578 sec/step)
```
I suppose that because of **NaN as a result of loss function it is not being trained**. Could anyone point me what could be wrong? (

So far I have tried:

- experimenting with different decay 
 
- experimenting with learning_rate values 

- loading model VGG_VOC0712_SSD_300x300_ft_iter_120000.ckpt 

none has helped

**Comment by burgetrm** [comment]
Solved 
1) Model was downloaded from:
`wget http://download.tensorflow.org/models/vgg_16_2016_08_28.tar.gz`
2) Unpacked to checkpoints directory
3) And the following command do the training
```
DATASET_DIR=./tfrecords
TRAIN_DIR=./log/
CHECKPOINT_PATH=./checkpoints/vgg_16.ckpt
python3 train_ssd_network.py \
    --train_dir=${TRAIN_DIR} \
    --dataset_dir=${DATASET_DIR} \
    --dataset_name=pascalvoc_2007 \
    --dataset_split_name=train \
    --model_name=ssd_300_vgg \
    --checkpoint_path=${CHECKPOINT_PATH} \
    --checkpoint_model_scope=vgg_16 \
    --checkpoint_exclude_scopes=ssd_300_vgg/conv6,ssd_300_vgg/conv7,ssd_300_vgg/block8,ssd_300_vgg/block9,ssd_300_vgg/block10,ssd_300_vgg/block11,ssd_300_vgg/block4_box,ssd_300_vgg/block7_box,ssd_3
    --trainable_scopes=ssd_300_vgg/conv6,ssd_300_vgg/conv7,ssd_300_vgg/block8,ssd_300_vgg/block9,ssd_300_vgg/block10,ssd_300_vgg/block11,ssd_300_vgg/block4_box,ssd_300_vgg/block7_box,ssd_300_vgg/bl
    --save_summaries_secs=60 \
    --save_interval_secs=600 \
    --weight_decay=0.0005 \
    --optimizer=adam \
    --learning_rate=0.001 \
    --learning_rate_decay_factor=0.94 \
    --batch_size=32

```

**Comment by zzks** [comment]
seems there are some codes not finished in one line, could you  complete these codes?
THX!
`--checkpoint_exclude_scopes=ssd_300_vgg/conv6,ssd_300_vgg/conv7,ssd_300_vgg/block8,ssd_300_vgg/block9,ssd_300_vgg/block10,ssd_300_vgg/block11,ssd_300_vgg/block4_box,ssd_300_vgg/block7_box,ssd_3
    --trainable_scopes=ssd_300_vgg/conv6,ssd_300_vgg/conv7,ssd_300_vgg/block8,ssd_300_vgg/block9,ssd_300_vgg/block10,ssd_300_vgg/block11,ssd_300_vgg/block4_box,ssd_300_vgg/block7_box,ssd_300_vgg/bl`
